client/src/components/AddVideo.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+
+function AddVideo({ onAddVideo }) {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [Error, setError] = useState("");
+
```

Comment:
you can also set 0 vote here for new videos, now when user add new video they cant vote for it becuase its undefined.